docs/book/api-docs/index.html

```
@@ -0,0 +1 @@
+# API Reference (rustdoc)
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)